### PR TITLE
Fixes for navigation and outline view

### PR DESCRIPTION
**Pull request by najadojo**
Outline shows negative size on compressed models #91
ERROR: Can no longer find document in editor #89

### CODE REVIEW

**Comment by emackey** [comment]
Thanks @najadojo.  Will get this merged soon.

I have some (mild) concerns about behavior on case-sensitive file systems, but currently don't have Mac or Linux handy to test.  @lexaknyazev I don't suppose you're in a position to test on such a system?  We have [build instructions](https://github.com/AnalyticalGraphicsInc/gltf-vscode/blob/master/CONTRIBUTING.md#developer-environment).

**Comment by emackey** [comment]
This works great on Windows, so any Mac/Linux users out there who find any trouble please file a new issue, thanks!  Merging.

**Comment by najadojo** [comment]
I tried this out on Linux and the path name is getting lowercased somewhere along the way.

File: src/dataUriTextDocumentContentProvider.ts [hunk 1]
```diff
@@ -139,14 +139,18 @@ export class DataUriTextDocumentContentProvider implements TextDocumentContentPr
...
     public async provideTextDocumentContent(uri: Uri): Promise<string> {
-        const filename = decodeURIComponent(uri.authority);
+        let fileName = decodeURIComponent(uri.authority);
         const query = querystring.parse<QueryDataUri>(uri.query);
         query.viewColumn = query.viewColumn || ViewColumn.Active.toString();
-        const document = vscode.workspace.textDocuments.find(e => e.fileName.toLowerCase() === filename.toLowerCase());
-        if (!document) {
-            return 'ERROR: Can no longer find document in editor: ' + filename;
+        let glTFContent: string;
+        const document = vscode.workspace.textDocuments.find(e => e.fileName.toLowerCase() === fileName.toLowerCase());
```
Comment (emackey) [inline src/dataUriTextDocumentContentProvider.ts]: If I recall correctly, that's why this is explicitly case-insensitive.  The `filename` comes from decoding `uri.authority`, which is lowercased.  Perhaps we need to move the filename out of `uri.authority` and over to `uri.path` which is case-sensitive.  Would need some new way to split the file path from the JSON path in that case.

Comment (najadojo) [reply src/dataUriTextDocumentContentProvider.ts]: Right I see... we could use the fragment or query string for the path... I'll see what I can do.

**Pull request merged**